app/src/main/java/com/jerboa/ui/theme/Color.kt

```diff
@@ -1061,3 +1061,9 @@ val Color.muted @Composable get() = this.copy(alpha = 0.5F)
 
 val CARD_COLORS @Composable
 get() = CardDefaults.cardColors(containerColor = MaterialTheme.colorScheme.inverseOnSurface)
+
+// TODO integrate into theme?
+val imageHighlight: Color
+    get() = Color(0xCCD1D1D1)
+val videoHighlight: Color
+    get() = Color(0xCCC20000)
```

Comment:
I'm not so sure about using existing Material color entries, for a few reasons:

- The colors need to be immediately distinctive between each other. 
- The colors need to be chosen specifically to show up properly over most thumbnails, and material theming doesn't intentionally provide that. 
- Boost has white, green, blue, and red. If we want to work up to that behavior we'll run out of slots.

If we want to integrate these into theme colors, we should find a way to extend Material colors IMO. Thoughts?

Reply:
Yes i agree, and i posted a link how to extend Material colors

Reply:
Added custom color capability with the theming system according to that article; let me know what you think. The biggest diversion I made was storing a reference to the default material theme rather than mirroring each and every single property; I think it's more future-proof this way, because if Material gets an update with more/less properties we won't have to change that mirroring solution. (Plus there are just a lot). 

Other than that it should be pretty straightforward.